packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1024,7 +1026,7 @@ export class ProjectService {
     async getResultsForChart(
         user: SessionUser,
         chartUuid: string,
-    ): Promise<Record<string, any>[]> {
+    ): Promise<{ rows: Record<string, any>[]; cacheMetadata?: CacheMetadata }> {
```

Comment:
any reason for cacheMetadata to be optional ?

Reply:
Nope. Will fix.